psutil/_psutil_aix.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2009, Giampaolo Rodola'. All rights reserved.
```

Comment:
Please also add your name. Also do the same in other new files (copy license and add your name).